### DISCUSSION TITLE
Customize save folders by type (Some difficulties)

### DISCUSSION THREAD

**Discussion by Rhodynoliaeth** [SignalFlagZ/sd-webui-civbrowser#63] [Q&A]
Hello, first of all, thank you for your work that makes this passion so enjoyable.
Then, I tried to apply your explanations and I must admit to being quite clumsy to apply them.

I am trying to change the default folder of Checkpoints in the settings and I have a symbolic link to another hard drive, so I tried to enter the following.
{
"Checkpoint": "Forge\webui\models\Stable-diffusion\Flux M4",
}

But unfortunately, when I test this change it offers me the default folder.

Do you have any advice please?

**Comment by SignalFlagZ** [comment]
I didn't notice the new discussion. Sorry for the late response.
There are two problems with your example. When you view the Checkpoint model, you will see messages in the console.
`CivBrowser: Check subfolder setting: Invalid \escape: line 2 column 21 (char 22)`
The first one is: 
The folder separator is "/", not "\\".
The second one is:
 Commas at the end of the last line violate json syntax.
The fix would be this:
```
{
"Checkpoint": "Forge/webui/models/Stable-diffusion/Flux M4"
}
```
It is recommended to check the json syntax using VS Code etc.
;)

**Comment by SignalFlagZ** [comment]
Stability Matrix has been updated so that the checkpoint folder is under the sd folder. This change causes the checkpoint save folder to be misaligned. 
To deal with this update, do the following:
`{"Checkpoint":"Stable-diffusion/sd"}`
